media/css/devreg/reviewers.styl

```diff
@@ -1092,6 +1092,19 @@ table.activity .error {
     }
 }
 
+.review-actions-testedon {
+    .formset {
+        margin-top: 1.5em;
+    }
+    a.add-row {
+        background: url("../../img/icons/news-feed-sprite.png") no-repeat 0 -1444px;
```

Comment:
Yeah, one of the problems about sprites are that the browser needs to hold the entire decompressed image in memory while the page is loaded, which is height x width bytes, regardless of how well it's compressed. Reducing requests are good, but you have to be pretty careful about how you do it. The best medium will often be in several smaller sprites.